tox.ini

```diff
@@ -174,7 +174,7 @@ deps =
     adapter_gunicorn-aiohttp03-py312: aiohttp==3.9.0rc0
     adapter_gunicorn-gunicorn19: gunicorn<20
     adapter_gunicorn-gunicornlatest: gunicorn
-    adapter_hypercorn-hypercornlatest: hypercorn
+    adapter_hypercorn-hypercornlatest: hypercorn<0.16
```

Comment:
Maybe we could add a comment here so we remember to come back and address the failures on the latest version?

Reply:
Good call.  Done!